core/src/main/java/org/apache/accumulo/core/clientImpl/mapreduce/lib/ConfiguratorBase.java

```diff
@@ -62,6 +63,8 @@
     IS_CONFIGURED, PRINCIPAL, TOKEN
   }
 
+  public static final String tokenCacheMnemonic = "tokenfile";
```

Comment:
Isn't this just the name of the tokenFile? This is less confusing to me:
`public static final String tokenFileName = "tokenfile";`